addons/point_of_sale/controllers/main.py

```diff
@@ -137,6 +137,7 @@ def invoice_request_screen(self, **kwargs):
             'errors': errors,
             'banner_error': " ".join(errors.values()),
             'form_values': form_values,
+            'ref': order_uuid,
```

Comment:
May I know what is this change for?

Reply:
yes it is no more required now
I will update it now